src/app/terms-landing/terms-landing.html

```diff
@@ -0,0 +1,171 @@
+<main class="form-page">
```

Comment:
Rename files to just `terms` no need for `-landing` at the end

Reply:
Done

Reply:
Resolved.